go/consensus/tendermint/apps/beacon/beapoch.go

```diff
@@ -0,0 +1,126 @@
+// Package beacon implements the combined beacon and epochtime
```

Comment:
This file is still named `beapoch.go`, should this just be moved under `beacon.go`?